src/platform/windows/device.rs

```diff
@@ -28,6 +28,7 @@ use crate::platform::windows::verify_dll_file::{
 /// A TUN device using the wintun driver.
 pub struct Device {
     pub(crate) tun: Tun,
+    #[allow(unused)]
     mtu: u16,
```

Comment:
Why leave this in the struct, unused?

Reply:
Sorry, I forget to delete it after testing